syncapi/routing/routing.go

```diff
@@ -42,6 +42,7 @@ func Setup(
 	r0mux := csMux.PathPrefix("/r0").Subrouter()
 
 	// TODO: Add AS support for all handlers below.
+	// asdf
 	r0mux.Handle("/sync", httputil.MakeAuthAPI("sync", userAPI, func(req *http.Request, device *userapi.Device) util.JSONResponse {
```

Comment:
I'm still having a lot of trouble trying to get Sytest to work. I'm seeing `Connection closed` in the fixture setup before a test can even get started.

 - macOS 10.15.7
 - postgres (PostgreSQL) 13.1

```
$ ./run-tests.pl -I Dendrite::Monolith -W ../dendrite/sytest-whitelist --client-log --stop-on-fail -vvvv
Running tests/10apidoc/01register.pl...
 # Starting test GET /register yields a set of flows
 # Starting fixture FIXTURE-7
 # Starting fixture HOMESERVER_0
 # Starting fixture FIXTURE-0
  Testing if: GET /register yields a set of flows...
Tests: 0 / 858 | OKSignature ok
subject=/CN=localhost
 # Starting fixture FIXTURE-3
 # Starting fixture FIXTURE-1
 # Fixture FIXTURE-1 now ready
 # Starting fixture FIXTURE-2
 # Fixture FIXTURE-2 now ready
 # Started test HTTPS Server at localhost:60549
  Testing if: GET /register yields a set of flows...
Tests: 0 / 858 | OKRequesting:
  POST https://localhost:60549/http_server_self_test HTTP/1.1
  Host: localhost
  Content-Length: 25
  Content-Type: application/json

  {"some_key":"some_value"}
 # Started test SMTP Server at localhost:60550
 # Fixture FIXTURE-3 now ready
 ** HOMESERVER_0: input fixture failed with Connection closed at ./run-tests.pl line 551.
 ** FIXTURE-7: input fixture failed with Connection closed at ./run-tests.pl line 551.
 ** Fixture failed with Connection closed at ./run-tests.pl line 720.
  Testing if: GET /register yields a set of flows...
Tests: 0 / 858 | OKReceived Request for POST /http_server_self_test:
  {"some_key":"some_value"}
 ** Destroying unresponded HTTP request to /http_server_self_test at /Users/eric/perl5/perlbrew/perls/perl-5.32.0/lib/site_perl/5.32.0/Net/Async/HTTP/Server/Protocol.pm line 53.
  Testing if: GET /register yields a set of flows... FAIL:
 | fixture failed - Connection closed
 +----------------------

1 tests FAILED
```

Reply:
🤷 This seems to work.

```
POSTGRES=1 ./run-tests.pl tests/31sync/09archived.pl -I Dendrite::Monolith -d ../dendrite/bin -W ../dendrite/sytest-whitelist --stop-on-fail
```